communication/remote.c

```diff
@@ -412,6 +426,15 @@ void remote_get_command_from_remote(remote_t* remote, control_command_t* control
 	controls->thrust 		= remote_get_throttle(remote);
 }
 
+void remote_get_gimbal_command_from_remote(remote_t* remote, control_command_t* controls)
```

Comment:
Can you use a `attitude_command_t` instead of the new field controls->gimbal_rpy ?